spatialpy/Solver.py

```diff
@@ -95,9 +104,14 @@ def compile(self, debug=False, profile=False):
 
         # Build the solver
         makefile = self.SpatialPy_ROOTDIR+'/build/Makefile'
-        cmd_list = ['cd', self.build_dir, '&&', 'make', '-f', makefile,
+        cmd_list = ['cd', self.core_dir, '&&', 'make', 'CORE', '-f',  makefile,
+            'ROOT="' + self.SpatialPy_ROOTPARAM+'"',
+            'ROOTINC="' + self.SpatialPy_ROOTINC+'"',
```

Comment:
ROOT and ROOTINC are in this list twice

Reply:
They belong to 2 different make calls

Reply:
one is for the cached core setup, the other is for the template